src/webgl/context-debug.js

```diff
@@ -58,7 +64,7 @@ export function getDebugContext(gl) {
 
   // Create a new debug context
   class WebGLDebugContext {}
-  const debugContext = WebGLDebug.makeDebugContext(gl, throwOnError, validateArgsAndLog);
+  const debugContext = global.WebGLDebug.makeDebugContext(gl, throwOnError, validateArgsAndLog);
```

Comment:
Nit: save `WebGLDebug` in a local const and use it below.

Reply:
I tried both ways, preferred it like this for some reason, but could have gone either way...